fetch.bs

```diff
@@ -2274,7 +2287,9 @@ is to return the result of <a>serializing a request origin</a> with <var>request
 
 <ol>
  <li><p>Let <var>newRequest</var> be a copy of <var>request</var>, except for its
- <a for=request>body</a>.
+ <a for=request>body</a> and <a for=request>id</a>.
+
+ <li><p>Set <var>newRequest</var>'s <a for=request>id</a> to a new unique string.
```

Comment:
Is this a UUID in practice? Do we just want to settle on that?

Reply:
It doesn't seem to be at the moment. I'd happily require a UUID here, but I feel like we don't quite have consensus on UUIDs everywhere yet.

Reply:
@annevk would you mind taking another look at this PR? Thanks!